### PR TITLE
Added basic asp:TextBox control support

### PR DESCRIPTION
**Pull request by comsemer**
## Description
* Control converters will now pick up asp:TextBox components
* Added support for main attributes/events used by asp:TextBox
* Added node template for single tag nodes
* Note that code-behind references to asp:TextBox components and other
advanced features are still not being handled, as is the case with the
other control converters

## Supplemental testing
* Added unit tests for new control converter

---
*By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.*

### CODE REVIEW

**Review by saikiranakula-amzn — APPROVED** [review]

File: tst/CTA.WebForms2Blazor.Tests/ControlConverters/TextBoxControlConverterTests.cs [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            var actualString = convertedNode.WriteTo();
+
+            Assert.AreEqual(expectedString, actualString);
+        }
+
+        [Test]
+        public void Convert2Blazor_Sets_ReadOnly_Attribute_Properly()
+        {
+            var htmlString = @"<asp:TextBox ID=""TestId"" Text=""Example Text"" ReadOnly=""true"">";
+            var expectedString = @"<input id=""TestId"" value=""Example Text"" readonly="""">";
```
Comment (jonlouie) [inline tst/CTA.WebForms2Blazor.Tests/ControlConverters/TextBoxControlConverterTests.cs:60]: Does the element created in `expectedString` function as expected? My understanding is that the `readonly` attribute does not take an assignment, for example: 
`<input id=""TestId"" value=""Example Text"" readonly>`

We may need to create a new `ViewLayerControlAttribute` to handle this case.

File: tst/CTA.WebForms2Blazor.Tests/ControlConverters/TextBoxControlConverterTests.cs [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            var actualString = convertedNode.WriteTo();
+
+            Assert.AreEqual(expectedString, actualString);
+        }
+
+        [Test]
+        public void Convert2Blazor_Sets_Disabled_Attribute_Properly()
+        {
+            var htmlString = @"<asp:TextBox ID=""TestId"" Text=""Example Text"" Enabled=""false"">";
+            var expectedString = @"<input id=""TestId"" value=""Example Text"" disabled="""">";
```
Comment (jonlouie) [inline tst/CTA.WebForms2Blazor.Tests/ControlConverters/TextBoxControlConverterTests.cs:73]: Same as above, but for the `disabled` attribute.

**Review by jonlouie — CHANGES_REQUESTED** [review]

File: tst/CTA.WebForms2Blazor.Tests/ControlConverters/TextBoxControlConverterTests.cs (hunk 1 above)
Comment (comsemer) [reply tst/CTA.WebForms2Blazor.Tests/ControlConverters/TextBoxControlConverterTests.cs:60]: I believe this format is also valid for boolean attributes, according to the html standard (https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attributes) the `readonly` attribute can be written as `readonly`, `readonly=readonly`, or `readonly=""`. If you want I can still make the change to handling boolean attributes the first way for stylistic reasons

**Review by jonlouie — APPROVED** [review]

File: tst/CTA.WebForms2Blazor.Tests/ControlConverters/TextBoxControlConverterTests.cs (hunk 1 above)
Comment (jonlouie) [reply tst/CTA.WebForms2Blazor.Tests/ControlConverters/TextBoxControlConverterTests.cs:60]: Awesome, sounds good. Should be fine as-is, thanks!

**Pull request merged**